menu.js

```diff
@@ -1,19 +1,152 @@
+    content2: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus repellendus mollitia saepe maxime vitae nostrum repellat eligendi amet voluptas modi repudiandae necessitatibus animi harum obcaecati at omnis laboriosam sapiente eum rem velit molestias',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Exploring the future of media in Facebook,s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    image: 'Images/Snapshoot Portfolio.png',
+    technologies: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+    content1: 'Lorem ipsum it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    content2: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus repellendus mollitia saepe maxime vitae nostrum repellat eligendi amet voluptas modi repudiandae necessitatibus animi harum obcaecati at omnis laboriosam sapiente eum rem velit molestias',
+  },
+  {
+    name: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: 'Images/Snapshoot Portfolio (1).png',
+    technologies: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+    content1: 'Lorem ipsum it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    content2: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus repellendus mollitia saepe maxime vitae nostrum repellat eligendi amet voluptas modi repudiandae necessitatibus animi harum obcaecati at omnis laboriosam sapiente eum rem velit molestias',
+  },
+];
+
```

Comment:
- [ ] Please make sure the information is visible of the pop ups, the images appear blurred and all the text does not appear, please see image below and kindly fix that. Please make sure all the information is displayed on the pop up. 
![reviews1](https://user-images.githubusercontent.com/65852621/211050212-12079a36-faf8-4e92-a285-f025e2e7c5a5.PNG)

![review2](https://user-images.githubusercontent.com/65852621/211050273-70163e8f-3c67-42b3-bd4b-89fd9e7ea7df.PNG)

Please make sure all the information appears on the pop ups and it should be responsive.